pkg/distributor/distributor.go

```diff
@@ -542,6 +545,13 @@ func (d *Distributor) cleanupInactiveUser(userID string) {
 	}
 }
 
+func (d *Distributor) cleanupInactiveGroupsForUser(userID, group string) {
```

Comment:
The name of the method should reflect what it's doing, not why it should be called:

```suggestion
func (d *Distributor) removeGroupMetricsForUser(userID, group string) {
```